assets/pages/oneoff-contributions/components/oneoffContributionsPayment.jsx

```diff
@@ -59,46 +77,16 @@ function mapStateToProps(state) {
   };
 }
 
-function mapDispatchToProps(dispatch: Dispatch<Action>) {
+function mapDispatchToProps(dispatch: Dispatch<*>) {
```

Comment:
Are you unable to use `Action` from `checkoutFormActions` as the type parameter here because `dispatch` is passed through to the component and used for other things?